hack/make/validate-test

```diff
@@ -11,7 +11,7 @@ unset IFS
 badFiles=()
 for f in "${files[@]}"; do
 	# skip check_test.go since it *does* use the testing package
-	if [ "$f" = "integration-cli/check_test.go" ]; then
+	if [ "$f" = "integration-cli/check_test.go" ] || [ "$f" = "integration-cli/utils_test.go" ]; then
```

Comment:
This is temporary (and as is pretty bad) to make sure the validation does not "mis-interpret" my unit tests with integration tests. Will have to find a cleaner solution though.